### ISSUE TITLE
"This chat is read only and you cannot post to it"

### ISSUE THREAD

**Issue by galenwp** [urbit/urbit#1998]
A while ago, I created a 'village' channel between me and one other person. Today, I went back to use it. I can post messages to it from the console but in the landscape ui I see:

<img width="601" alt="Screen Shot 2019-11-27 at 11 03 32 AM" src="https://user-images.githubusercontent.com/1467211/69752238-a50fb480-1105-11ea-96d2-ddec0c1a90d7.png">

The other person in the chat can't post at all. Anything I can do to diagnose?

**Comment by ngzax** [comment]
I just came across this myself. In my case I initially incorrectly created the channel on my star, ~darlur, as a journal which is read-only so I deleted it and created it again properly as a channel. However, i has already subscribed as ~winter-paches. I left and re-joined and now I can chat via the chat-cli app but not landscape.

<img width="930" alt="Screen Shot 2020-02-01 at 2 20 50 PM" src="https://user-images.githubusercontent.com/87197/73597881-14879300-44ff-11ea-825d-112044227ddf.png">
